spt/utils/ent_utils.cpp

```diff
@@ -318,31 +320,8 @@ namespace utils
 
 		while (i < argSize && entries < maxEntries)
 		{
-			bool readEntityIndex = false;
-
-			// Read entity index
-			if (args[i] == entSep)
-			{
-				++i;
-				readEntityIndex = true;
-			}
-			// Also read if at the start of the string. Note: does not increment the index since we're at the start of the string.
-			else if (i == 0)
-			{
-				readEntityIndex = true;
-			}
-			else if (args[i] == sep)
-			{
-				++i;
-			}
-			else // error occurred
-			{
-				i = argSize;
-				arr = (wchar_t*)L"error occurred in parsing";
-				entries = 1;
-				break;
-			}
-
```

Comment:
Any particular reason why the error check (else branch in the old code) was removed?